workers/unity/Packages/com.improbable.gdk.core/Commands/WorldCommands.cs

```diff
@@ -14,34 +14,34 @@ internal static void AddWorldCommandRequesters(World world, EntityManager manage
         {
             var createEntitySender = new CreateEntity.CommandSender
             {
-                Handle = CreateEntity.RequestsProvider.Allocate(world)
+                Handle = CreateEntity.RequestsProvider.Allocate(world),
+                RequestsToSend = new List<CreateEntity.Request>()
```

Comment:
Is this right? The handle needs to point to this list so I'm sure this is correct.

Reply:
I don't understand - I can't see how `Handle` is related to `RequestsToSend` et al, at all.
If they are somehow, we need to fix our API, because that is guaranteed to be misused.

Reply:
This is a workaround for blittable (and lack there of). To enable any lists in an `IComponentData`, we have a handle that points to it somewhere else. `RequestToSend` is a property that fetches that list using the Handle.

This API needs some tweaking for safety - but something similar to it is unavoidable until we have a replacement for lists and maps and strings.

Reply:
Wow, that's really fragile.
Let me see what I can do.